pages/Documents/DeveloperTools/LivePersonFunctions/ToolbeltDocumentation/toolbelt.md

```diff
@@ -24,6 +24,8 @@ indicator: both
 - [SFClient](#sfclient)
 - [SecretClient](#secretclient)
 
+{: .important}
+To ensure that infrastructure security doesn’t block communication from your function to the Context service, **ContextServiceClient** needs to be whitelisted with correct domain. 
```

Comment:
Please make sure to add the correct link. I was not able to look it up completely.


```suggestion
Please be aware that `ContextServiceClient` **needs** to have the correct (zone-specific) domain added to the [allowlist](liveperson-functions-foundations-features.html#xy). As this is not done by default.

```

Also, always make sure always have a new line between `{: .important}` and the next content